infra/README.md

```diff
@@ -0,0 +1,139 @@
+Default region name [None]: us-east-2
+Default output format [None]: json
+```
+**Using the above command will create a [default] profile.**  
+```
+$ aws configure --profile dev
+AWS Access Key ID [None]: AKIAIOSFODNN7EXAMPLE
+AWS Secret Access Key [None]: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+Default region name [None]: us-east-2
+Default output format [None]: json
+```
+**Using the above command will create a [dev] profile.**  
+
+### References:
+
+- [Configuration basics][1]
+- [Named profiles for the AWS CLI][2]
+- [Configuration and credential file settings][3]
+
+[1]: https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html
+[2]: https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html
+[3]: https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html
+
+## Basic Terraform Commands 
+
+&nbsp;&nbsp;The `terraform init` command is used to initialize a working directory containing Terraform configuration files. This is the first command that should be run after writing a new Terraform configuration or cloning an existing one from version control. It is safe to run this command multiple times.
+
+&nbsp;&nbsp;The `terraform plan` command creates an execution plan, which lets you preview the changes that Terraform plans to make to your infrastructure. By default, when Terraform creates a plan it:
+
+- Reads the current state of any already-existing remote objects to make sure that the Terraform state is up-to-date.
+- Compares the current configuration to the prior state and noting any differences.
+- Proposes a set of change actions that should, if applied, make the remote objects match the configuration.
+
+&nbsp;&nbsp;The `terraform apply` command executes the actions proposed in a Terraform plan.
+
+<span style="color:red"> ---- DANGER ----</span>\
+In a non-development environment comment out a resource to destroy, destroy should only be used as a way to cleanup a development environment. e.g. a developers workspace after they are done with it.
+
+&nbsp;&nbsp; The `terraform destroy` command is a convenient way to destroy all remote objects managed by a particular Terraform configuration.\
+
+For more information about terraform commands follow the link below:
+
+- [Basic CLI Features](https://www.terraform.io/cli/commands)
+
+## Terraform Backend
+
+### infra/bootstrap/account
+
+1. Rename bootstrap/account, account directory to the name of the aws account alias or account id where this infrastructure will be hosted.
+2. Customize the variables in locals{} at the top of main.tf to match the desired deployment setup.
+3. Open a terminal and cd into the infra/bootstrap/account directory and run the following commands:
+    - terraform init
+    - terraform plan
+    - terrafrom apply
+4. Uncomment out the backend "s3" {} block, fill in the appropriate information from outputs and repeate step `3.` to switch from local to remote backend.
```

Comment:
oh interesting! didn't think to do it this way on PFML 😛 guessing terraform doesn't get confused about running `terraform destroy` and destroying it's own tf state bucket along the way.

Reply:
Oh interesting, is this a better way to do it? Or should the remote backend be managed separately?

Reply:
If a developer were to come in a few months later, somehow CD into the bootstrap/account and then run a `terraform destroy` It would destroy the backend, then throw an error stating it cannot find the backend. I created a
```
  lifecycle {
    prevent_destroy = true
  }
```

on the s3 buckets created to host state files, just in case that scenario were to occur.